src/globals.ts

```diff
@@ -0,0 +1,48 @@
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+import Command from '@oclif/command';
+import { Specification } from './models/SpecificationFile';
+
+const GreenLog = chalk.hex('#00FF00');
+const OrangeLog = chalk.hex('#FFA500');
+const CHOKIDAR_CONFIG = {
+  // awaitWriteFinish: true // Used for large size specification files.
+
+};
+const ConditionalNestedLogs = {
+  logOnAutoDisable: (docVersion: 'old' | 'new' | '' = '') => docVersion === '' ? (' ' + 'AsyncAPI' + ' ') : (` ${docVersion} `)
+};
+const WATCH_MESSAGES = {
+  logOnStart: (filePath: string) => console.log(GreenLog(`Watching AsyncAPI file at ${filePath}\n`)),
+  logOnChange: (handlerName: string) => console.log(OrangeLog(`Change detected, running ${handlerName}\n`)),
+  logOnAutoDisable: (docVersion: 'old' | 'new' | '' = '') => console.log(OrangeLog(`Watch mode for${ConditionalNestedLogs.logOnAutoDisable(docVersion)}file was not enabled.`), OrangeLog('\nINFO: Watch works only with files from local file system\n'))
+};
+
+let IS_CHOKIDAR_INSTANCE_RUNNING = false;
+
+export type specWatcherParams = {
+  spec: Specification,
+  handler: Command,
+  handlerName: string,
+  docVersion?: 'old' | 'new' | ''
```

Comment:
I think that signature for that field should be:

```suggestion
  docVersion?: 'old' | 'new';
```

because optional field say that it can be also `undefined`, empty string is unnecessary, because it can be treated as undefined.